TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -455,7 +455,13 @@ def analyze(self,event):
       subfile.close()
       print "Saved condor submit file to %s" % options.subfile
       if not options.pretend:
-         os.system("condor_submit "+options.subfile)
+          if options.batch_name == "CMD:":
+              batch_name = " -batch-name '"+options.batch_name+options.runner+"'"
+          else:
+              batch_name = " -batch-name '"+options.batch_name+"'"
+          condor_submit_command = "condor_submit "+options.subfile+batch_name
+          print "Running:", condor_submit_command
+          os.system(condor_submit_command)
```

Comment:
These lines here modifies the submitting in order to split the jobs like below:
OWNER  BATCH_NAME                SUBMITTED   DONE   RUN    IDLE  TOTAL JOB_IDS
kpanos   dataWorkflow-2018-data     9/19 11:22            _      8      _      8 2212679.0-7
kpanos   dataWorkflow-2017-data     9/19 11:27            _      6      _      6 2212711.0-5
kpanos   dataWorkflow-2016-data     9/19 11:39            _      2      _      2 2212743.0-1